packages/privacy-toolset/src/do-not-sell-dialog/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { FormToggle } from '@wordpress/components';
+
+import './styles.scss';
+
+export type DoNotSellDialogProps = {
+	content: {
+		title: string;
+		longDescription: ( string | React.ReactNode )[];
+		toggleLabel: string;
+		closeButton: string;
+	};
+	onClose: () => void;
+	isActive?: boolean;
+	onToggleActive: ( isActive: boolean ) => void;
+};
+export const DoNotSellDialog = ( {
+	content,
+	onClose,
+	isActive,
+	onToggleActive,
+}: DoNotSellDialogProps ) => (
+	<div className="do-not-sell">
```

Comment:
I should have pointed this out on the cookie banner as well, but I think an aria `role` of `dialog` would be useful here. And perhaps some other aria attributes as well. My reference can be [seen here](https://techservicesillinois.github.io/accessibility/examples/notice/index.html)

Reply:
That's a good idea, I think it will be worth including aligning do not sell and cookie banners with aria recommendations. I'll include that as a follow-up task on privacy work. This change I'll address in this PR though.